temperature/router.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from dependencies import get_db
+from temperature import crud, schemas
+
+router = APIRouter()
+
+
+@router.get("/temperatures/", response_model=list[schemas.Temperature])
```

Comment:
Consider using `List[schemas.Temperature]` as a response model type hint for `get_temperatures` to ensure compatibility with Python versions prior to 3.9. Alternatively, you can import `List` from `typing` and use `List[schemas.Temperature]`.